samples/MASMessagingSample/build.gradle

```diff
@@ -33,10 +33,13 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    testCompile 'junit:junit:4.12'
     compile "com.android.support:recyclerview-v7:$rootProject.ext.recyclerViewVersion"
     compile "com.android.support:design:$rootProject.ext.designVersion"
+    compile 'pl.droidsonroids.gif:android-gif-drawable:1.2.7'
```

Comment:
So I know in your email you wrote about the launch screen being a bit overkill but that we wouldn't be demo-ing it to sales...are you still intending on checking in this addition? :P And if so, do we need to get a TPSR for this library?